Jenkinsfile

```diff
@@ -191,6 +207,31 @@ dockerBuild {
                     sh 'travis/test-upgrade-convert.sh v230-nondocker ${HOME}/workspace'
                 }
             }
+            stage('Publish') {
+                parallel 'Push to Docker hub': {
+                    withCredentials([string(credentialsId: 'docker-hub', variable: 'DOCKER_PASSWORD')]) {
+                        env.DOCKER_PASSWORD = DOCKER_PASSWORD
+                        sshagent (credentials: ['c2c-infra-ci']) {
+                            sh 'travis/publish-docker'
+                        }
+                    }
+                }, 'Push to Transifex': {
+                    branch = sh(returnStdout: true, script: 'git rev-parse --abbrev-ref HEAD')
+                    if (branch == MAIN_BRANCH) {
+                        withCredentials([string(credentialsId: 'transifex-sbrunner', variable: 'TRANSIFEX_PASSWORD')]) {
+                            sh 'echo "[https://www.transifex.com]" >> ~/.transifexrc'
```

Comment:
Does the .transifexrc exists already ? 
If no, and to be sure the file is clean, you should use > on the first redirection instead of happening.